### PR TITLE
openzfs-dev 2.1.99

### PR DESCRIPTION
**Pull request by yurikoles**
Signed-off-by: Yurii Kolesnykov <root@yurikoles.com>

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [ ] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

### CODE REVIEW

**Comment by yurikoles** [comment]
Unrelated RAID kext error is expected.

File: Casks/openzfs-dev.rb [hunk 1]
```diff
@@ -1,20 +1,12 @@
...
+    version "2.1.99,369"
+    sha256 "932db021cddc7de34aa9d50cdad723b0b437dfdf2161150ca416fc68cad8e1c9"
+    pkg "OpenZFSonOsX-#{version.csv.first}-EL.CAPITAN-10.11.pkg"
   else
-    version "2.1.0rc4,338"
-    sha256 "9e25ff3d7701b6f84128714b0f5ef81b774fca30cd80103b67430775c98e5805"
-    pkg "OpenZFSonOsX-#{version.csv.first}-Big.Sur-11-arm64.pkg"
+    version "2.1.99,368"
+    sha256 "c64f103b696218202df7f758cc9d4c9995f56b2108fa072eb4c03aef9457219c"
+    pkg "OpenZFSonOsX-#{version.csv.first}-Catalina-10.15.pkg"
```
Comment (bevanjkay) [inline Casks/openzfs-dev.rb:9]: @yurikoles Do you know if this package works on Big Sur and Monterey?

Comment (yurikoles) [reply Casks/openzfs-dev.rb:9]: @lundman, the author said that this package is for Catalina and up.

Comment (yurikoles) [reply Casks/openzfs-dev.rb:9]: https://openzfsonosx.org/forum/viewtopic.php?f=20&t=3677

>If anyone wants to test it out, there is a .pkg installer for Catalina (and newer versions).

**Comment by yurikoles** [comment]
I amended commit with correct cask name.

**Comment by yurikoles** [comment]
WAT? The very same commit diff with different message failed test?

File: Casks/openzfs-dev.rb (hunk 1 above)
Comment (yurikoles) [reply Casks/openzfs-dev.rb:9]: And, yes, I can confirm that it indeed works on Monterey.

**Comment by yurikoles** [comment]
Here is proof, [successful job](https://github.com/Homebrew/homebrew-cask-versions/actions/runs/1931773254) for this patch.

**Pull request merged**